lighthouse-extension/test/extension-test.js

```diff
@@ -76,6 +76,7 @@ describe('Lighthouse chrome extension', function() {
     });
 
     if (lighthouseResult.exceptionDetails) {
+      console.error(lighthouseResult); // eslint-disable-line no-console
```

Comment:
will you add a comment explaining why logging here?

Reply:
done